pkg/controller/registry/resolver/resolver.go

```diff
@@ -548,20 +548,32 @@ func (r *SatResolver) sortChannel(bundles []*Operator) ([]*Operator, error) {
 		}
 	}
 
-	if len(headCandidates) != 1 {
-		// TODO: more context in error
-		return nil, fmt.Errorf("found more than one head for channel")
+	if len(headCandidates) > 1 {
+		var p string
+		for _, v := range headCandidates {
+			if v.sourceInfo.Package != p {
+				p = v.sourceInfo.Package
+			} else {
+				// if find multi heads in the same package, return error
+				return nil, fmt.Errorf("found more than one head for package: %s, please check the `replaces` or `skipRange` fields of the operator bundles", p)
+			}
+		}
+
+	} else if len(headCandidates) < 1 {
+		return nil, fmt.Errorf("Not found the head for channel")
```

Comment:
nit: can this error message be `head of channel not found`?

Reply:
Sure, thanks!